lib/tasks/multitenant_db_merge.rake

```diff
@@ -0,0 +1,344 @@
+desc 'Merge an ext daria db into main db'
+task multitenant_db_merge: :environment do
+  raise 'SET MIGRATION_FUND_ID ENV VAR' if not ENV['MIGRATION_FUND_ID']
+  raise 'SET MIGRATION_DB_URL ENV VAR' if not ENV['MIGRATION_DB_URL']
```

Comment:
My plan is to pretty much run this as a rake on heroku dynos, a la `heroku run rails multitenant_db_merge -a ...` after setting particular env vars. Not worried about spillover - to avoid this we'll write the playbook to explicitly unset the env vars, I think.